payment_redsys/controllers/main.py

```diff
@@ -1,6 +1,6 @@
-# © 2016-2017 Sergio Teruel <sergio.teruel@tecnativa.com>
-# © 2019 Ignacio Ibeas <ignacio@acysos.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# Copyright 2016-2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# Copyright 2019 Ignacio Ibeas <ignacio@acysos.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 import logging
 import pprint
```

Comment:
Remove this

Reply:
What about the debate we had about this?

Reply:
Any news about this? Is the only thing preventing to be merged. Remember that I prefer to remove that verbose logging

Reply:
If I try other methods, I catch an error. So I think that it's important

Reply:
Errors shouldn't be "catched" this way, as this is only informative. You must:

- Or remove it
- Or change logging level to debug

Reply:
Okey, I remove it